alf/experience_replayers/replay_buffer.py

```diff
@@ -68,6 +69,11 @@ def __init__(self,
             num_environments (int): total number of parallel environments
                 stored in the buffer.
             max_length (int): maximum number of time steps stored in buffer.
+            num_earliest_frames_ignored (int): ignore the earlist so many frames
+                when sample from the buffer. This is typically required when
```

Comment:
"ignore the earlist so many frames when sample from the buffer." is a bit ambiguous, especially when combined with the espisodic_info comment below.
"ignore the earliest so many frames in the buffer, when sampling."  is more clear.